changelog/20.0/20.0.0/summary.md

```diff
@@ -28,6 +29,12 @@ This makes reparenting in Vitess resilient to client errors, and prevents Planne
 
 In order to preserve the old behaviour, the users can set the flag back to `0 seconds` causing open transactions to never be shutdown, but in that case, they run the risk of PlannedReparentShard calls timing out.
 
+#### <a id="unmanaged-tablet"/> New `unmanaged` Flag
+
+New flag `--unmanaged` has been introduced in this release to make it easier to flag unmanaged tablets. It also runs validations to make sure the unmanaged tablets are configured properly.
+
+Starting this release, all unmanaged tablets should specify this flag.
```

Comment:
People can still not specify this, and just specify the existing set of flags, correct?
We should make this language stronger, and say that we recommend setting this flag so that we can validate all the related flags.

Reply:
I was thinking about this again, and I believe we should deprecate `--disable_active_reparents` along with adding this flag.
For backwards compatibility, we should deprecate `--disable_active_reparents` in v20, and delete it in v21.
The internal value of `mysqlctl.DisableActiveReparents` should be the OR of `--unmanaged` and `--disable_active_reparents` in v20, and just the value of `--unmanaged` in v21.

Reply:
`disable_active_reparents` is used in vtctldclient as well to prevent that specific instance from running reparent commands. I don't think we can get rid of that. We could rename the flag on vtctld too?

Reply:
It should be fine to leave that flag on vtctld. Having the same flag name on vttablet and vtctld was always confusing. I'm not sure if the two are sharing code which might make this separation more complicated, though.

Reply:
Done 😎 I made the required changes and deprecated the flag only on the required binaries.